nemo/collections/asr/modules/conv_asr.py

```diff
@@ -239,6 +239,16 @@ def input_example(self):
         input_example = torch.randn(bs, self._feat_in, seq).to(next(self.parameters()).device)
         return tuple([input_example])
 
+    def _prepare_for_export(self):
+        m_count = 0
+        for m in self.modules():
+            if type(m).__name__ == "MaskedConv1d":
```

Comment:
I would agree with this as well for purposes of type checking/soundness, the string matching by type name came from the original implementation. 

Otherwise even worse would be scenario where the class name does not change but the name of `use_mask` changes  such that the conditional would be triggered but the `setattr(m, "use_mask", False)` would throw an assertion